.github/workflows/release-docker.yml

```diff
@@ -65,54 +51,28 @@ jobs:
   build_and_push:
     name: Build images and push to GHCR
     runs-on: ubuntu-latest
-    needs: [setup_image_builder, get_version_meta]
+    needs: [setup_image_builder, version_metadata]
     steps:
       - name: Get docker build arg
         id: build_arg
         run: |
-          BUILD_ARG_NAME=$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g')
-          BUILD_ARG_VALUE=$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g')
-          echo "build_arg_name=$BUILD_ARG_NAME" >> $GITHUB_OUTPUT
-          echo "build_arg_value=$BUILD_ARG_VALUE" >> $GITHUB_OUTPUT
+          BUILD_ARG_NAME=$(echo ${{ inputs.package }} | sed 's/\-/_/g')
+          BUILD_ARG_VALUE=$(echo ${{ inputs.package }} | sed 's/postgres/core/g')
+          echo "name=$BUILD_ARG_NAME" >> $GITHUB_OUTPUT
+          echo "value=$BUILD_ARG_VALUE" >> $GITHUB_OUTPUT
 
-      - name: Log in to the GHCR
+      - name: Log in to GHCR
         uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push MAJOR.MINOR.PATCH tag
-        uses: docker/build-push-action@v5
-        with:
-          file: docker/Dockerfile
-          push: True
-          target: ${{ github.event.inputs.package }}
-          build-args: |
-            ${{ steps.build_arg.outputs.build_arg_name }}_ref=${{ steps.build_arg.outputs.build_arg_value }}@v${{ github.event.inputs.version_number }}
-          tags: |
-            ghcr.io/dbt-labs/${{ github.event.inputs.package }}:${{ github.event.inputs.version_number }}
-
-      - name: Build and push MINOR.latest tag
-        uses: docker/build-push-action@v5
-        if: ${{ needs.get_version_meta.outputs.minor_latest == 'True' }}
-        with:
-          file: docker/Dockerfile
-          push: True
-          target: ${{ github.event.inputs.package }}
-          build-args: |
-            ${{ steps.build_arg.outputs.build_arg_name }}_ref=${{ steps.build_arg.outputs.build_arg_value }}@v${{ github.event.inputs.version_number }}
-          tags: |
-            ghcr.io/dbt-labs/${{ github.event.inputs.package }}:${{ needs.get_version_meta.outputs.major }}.${{ needs.get_version_meta.outputs.minor }}.latest
-
-      - name: Build and push latest tag
+      - name: Build and push `${{ inputs.package }}:${{ tag }}`
```

Comment:
This tag doesn't have a value.

Reply:
Nice catch. I started with a `strategy` to loop through the tags, then realized I could supply them as a comma delimited string, removed `strategy`, but never updated this. I'll fix it.